.github/workflows/release.yaml

```
@@ -5,65 +5,48 @@ on:
     tags:
       - '*'
 
+permissions:
+  contents: write
```

need to re-add packages permission to publish images
```suggestion
  contents: write
  packages: write
```